examples/examples.cjsx

```diff
@@ -80,7 +82,9 @@ module.exports = React.createClass
         strokeColor='green'
         strokeWidth='5px'
         interpolate='none'
-        circleDiameter=10 />
+        circleDiameter=10
+        viewBox="0 0 200 40"
```

Comment:
You are correct, my mistake.

On Wednesday, 22 July 2015, Kyle Mathews <notifications@github.com> wrote:

> In examples/examples.cjsx
> <https://github.com/KyleAMathews/react-sparkline/pull/9#discussion_r35124043>
> :
>
> > @@ -80,7 +82,9 @@ module.exports = React.createClass
> >          strokeColor='green'
> >          strokeWidth='5px'
> >          interpolate='none'
> > -        circleDiameter=10 />
> > +        circleDiameter=10
> > +        viewBox="0 0 200 40"
>
> viewBox shouldn't be a prop here correct since you're dynamically
> generating it from the width/height props?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KyleAMathews/react-sparkline/pull/9/files#r35124043>.
>


-- 
James Gregory

Tel: +61 (0) 411 619 513
Website: http://jagregory.com
Twitter: @jagregory <http://twitter.com/jagregory>